proof/invariant-abstract/AARCH64/ArchRetype_AI.thy

```diff
@@ -274,6 +271,7 @@ locale retype_region_proofs_arch
 
 context retype_region_proofs begin
 
+(* FIXME arch_split: is there any way to optimise this interpretation out? we can't nest contexts *)
 interpretation Arch .
```

Comment:
This is a bit funky. Either `retype_region_proofs` is supposed to be generic or not. There is `retype_region_proofs_arch` just above, so you'd think `retype_region_proofs` is the generic one, in which case it shouldn't do `interpretation Arch` and the lemmas should instead be inside `retype_region_proofs_arch`. 

Changing it probably breaks something down the line, but it looks like an abstraction violation to me. Definitely good to put a FIXME here.

Reply:
This one is beyond me, I think I'll leave it as a FIXME.